CHANGELOG.md

```diff
@@ -12,11 +12,3 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Added schema testing to validate SQL operations made by the input tool
 - Added the first version of the input tool module that opens database sessions for Swell workflow tasks
 - Added `CODEOWNERS` file
-
```

Comment:
You can keep these lines in here. They are just "section placeholders" so that if someone Adds something or Changes something, they can put it under the right section.

Reply:
Thanks Matt, good to know!